src/pages/GettingStarted/GettingStarted.messages.ts

```diff
@@ -0,0 +1,11 @@
+export const Messages = {
+  addContent: 'Add Content',
+  addOrganization: 'Add Organization',
+  readMore: 'Read More',
+  createOrganizationDescription: 'Creating your organization is going to allow you to invite others.',
```

Comment:
others what? Members? Organizations?

Reply:
```suggestion
  createOrganizationDescription: 'Creating your organization is going to allow you to invite other members.',
```

Maybe something like this then? Seems easier to understand to me

Reply:
Agreed 👍